### ISSUE TITLE
How to recover address after reinstall OS?

### ISSUE THREAD

**Issue by maximum733** [vaklinov/hush-swing-wallet-ui#9]
Hi there! i have a question about address, for example i have 5 hush in my wallet t1c77wPdrgGT7bb7t4ka1aDe1pCZ2uwrjXh and this wallet i use hush swing wallet ui for windows, so how to recover my hush coin at t1c77wPdrgGT7bb7t4ka1aDe1pCZ2uwrjXh address if i reinstall OS?

**Comment by vaklinov** [comment]
The HUSH is located inside the wallet file `wallet.dat`. You could copy it manually (when the GUI wallet is **not** running) or use the menu Wallet >> Backup. It is important to regularly back up your `wallet.dat`.

**Comment by maximum733** [comment]
So where is wallet.dat located if im not using nemu Wallet-Backup?

**Comment by vaklinov** [comment]
The `wallet.dat` location is shown in the lower left status field of the wallet, example:
https://raw.githubusercontent.com/vaklinov/hush-swing-wallet-ui/master/docs/HUSHSwingWalletUIWin.png

**Comment by maximum733** [comment]
Oh i see, thanks bro!

**Comment by sifotes** [comment]
Do I need to do anything else to move my wallet to a new computer? Ran the hush swing wallet on a new computer, waiting for the proving key to download, closed the wallet, copied the overwrote wallet.dat with the original file. When I run the new copy of the swing wallet, it does not show any of the addresses I used to have.

**Comment by vaklinov** [comment]
If you replace/overwrite the `wallet.dat` (while GUI wallet is stopped) you should see the addresses from the system where the `wallet.dat` came from. One issue may be that the blockchain in not yet synchronized 100% on the new system! Does the wallet still fail to show the addresses after blockchain gets synchronized 100% ?

**Comment by sifotes** [comment]
Yes. It has finished syncing but the previous addresses are not showing.

**Comment by vaklinov** [comment]
So if the wallet still does not show the addresses/balances after replace/overwrite of the wallet.dat (all this is very strange :-[ ), you may try the procedure of export/import private keys from the "Wallet" menu. The wallet need not be stopped during this procedure.
